runtime/lua/vim/filetype/detect.lua

```diff
@@ -1166,13 +1166,14 @@ function M.sh(path, contents, name)
       vim.b[b].is_bash = nil
       vim.b[b].is_sh = nil
     end
-  elseif vim.g.bash_is_sh or matchregex(name, [[\<bash\>]]) or matchregex(name, [[\<bash2\>]]) then
+  elseif vim.g.bash_is_sh or matchregex(name, [[\<bash|bash2\>]]) then
```

Comment:
```suggestion
  elseif vim.g.bash_is_sh or matchregex(name, [[\<\(bash|bash2\)\>]]) then
```

Reply:
That teaches me to blindly rely on `make oldtest`...